### PR TITLE
Updated Yarn CLI specs

### PR DESCRIPTION
**Pull request by shoeblabs**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Spec Update

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Additional info:**
- Added description in all of the root subcommands.
- Few sub commands details are pending.

### CODE REVIEW

File: specs/yarn.js [hunk 1]
```diff
@@ -1,3 +1,52 @@
...
+        if (out.trim() == "") {
+            return []
+        }
+
+        try {
+            let package = JSON.parse(out)
+            let dependencyScripts = package["dependencies"]
+            let devDependencyScripts = package["devDependencies"]
+            if (dependencyScripts || devDependencyScripts) {
+                return [...Object.keys(dependencyScripts), ...Object.keys(devDependencyScripts)]
```
Comment (rolandtshen) [inline specs/yarn.js:15]: What gets returned here needs to be formatted as a [suggestion object](https://withfig.com/docs/autocomplete/api#suggestion-object). I think having these keys as the name, and adding a little 📦 icon would be perfect.

Comment (rolandtshen) [reply specs/yarn.js:15]: Everything else looks great, ready to merge after this!

**Review by falky97 — APPROVED** [review]
Hey - thanks so much for adding this! Great additions to the spec, especially the generator.
Two notes: 
1. please always add error logging to any try catch statement
2. we can't assume that a package.json has a dependencies or devDependencies key... When I tested it this wasn't the case and your generator didn't work at all because we weren't accounting for it. Next time try and test on a few more types before merging! 
```
      const dependencyScripts = packageContent["dependencies"] || {};
      const devDependencyScripts = packageContent["devDependencies"] || {};
```

Regardless, still great for your first generator

**Comment by shoeblabs** [comment]
> Hey - thanks so much for adding this! Great additions to the spec, especially the generator.
> Two notes:
> 
> 1. please always add error logging to any try catch statement
> 2. we can't assume that a package.json has a dependencies or devDependencies key... When I tested it this wasn't the case and your generator didn't work at all because we weren't accounting for it. Next time try and test on a few more types before merging!
> 
> ```
>       const dependencyScripts = packageContent["dependencies"] || {};
>       const devDependencyScripts = packageContent["devDependencies"] || {};
> ```
> 
> Regardless, still great for your first generator

Thanks for the suggestion 👍

**Pull request merged**